src/style.css

```diff
@@ -1 +1,74 @@
-/* Styles go here */
+
+html {
+  margin: 0;
+  font-family: Roboto, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  margin: 0;
+  display: flex;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  align-items: center;
+  max-width: 1200px;
+  padding-left: 50px;
```

Comment:
make the same paddings at the both sides of header, do not specify 
`.nav {
  padding-right: 50px;
}`